docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "2"
+services:
+  db:
+    image: "postgres:9.5"
+
+  web:
```

Comment:
I might rename this as "api" , web typically refers to all the components from the askdarcel-web project.